javascript-light-bulb/main.js

```diff
@@ -0,0 +1,21 @@
+// query the dom for both the container and the lightbulb
+// add an a click event listener onto the lightbulb div
+// add an event handler that:
+//    checks if lightbulb is on or off
+//        if off, change classes of both lightbulb and container to on
+//        if on, turn both classes to off
+
+var $container = document.querySelector('#container');
+var $lightbulb = document.querySelector('#lightbulb');
+
+function lightSwitch(event) {
+  if ($lightbulb.className === 'bulb-off') {
```

Comment:
Instead of basing your functionality around the current classes on the DOM element, I would use a variable to keep track of if your lightbulb is "on" or "off". A boolean value would probably be a good choice. Remember what we discussed in Q&A, yesterday, we want all JavaScript functionality to not be reliant on data stored in the DOM as much as possible, you should have the information you need to do your work all centralized in your JavaScript..